aya/src/maps/info.rs

```diff
@@ -0,0 +1,410 @@
+    #[doc(alias = "BPF_MAP_TYPE_PERCPU_CGROUP_STORAGE")]
+    #[doc(alias = "BPF_MAP_TYPE_PERCPU_CGROUP_STORAGE_DEPRECATED")]
+    PerCpuCgroupStorage = bpf_map_type::BPF_MAP_TYPE_PERCPU_CGROUP_STORAGE as isize,
+    /// A Queue map type. See [`Queue`](super::queue::Queue) for the map implementation.
+    ///
+    /// Introduced in kernel v4.20.
+    #[doc(alias = "BPF_MAP_TYPE_QUEUE")]
+    Queue = bpf_map_type::BPF_MAP_TYPE_QUEUE as isize,
+    /// A Stack map type. See [`Stack`](super::stack::Stack) for the map implementation.
+    ///
+    /// Introduced in kernel v4.20.
+    #[doc(alias = "BPF_MAP_TYPE_STACK")]
+    Stack = bpf_map_type::BPF_MAP_TYPE_STACK as isize,
+    /// A Socket-local Storage map type.
+    ///
+    /// Introduced in kernel v5.2.
+    #[doc(alias = "BPF_MAP_TYPE_SK_STORAGE")]
+    SkStorage = bpf_map_type::BPF_MAP_TYPE_SK_STORAGE as isize,
+    /// A Device Hash Map type. See [`DevMapHash`](super::xdp::DevMapHash) for the map
+    /// implementation.
+    ///
+    /// Introduced in kernel v5.4.
+    #[doc(alias = "BPF_MAP_TYPE_DEVMAP_HASH")]
+    DevMapHash = bpf_map_type::BPF_MAP_TYPE_DEVMAP_HASH as isize,
+    /// A Struct Ops map type.
+    ///
+    /// Introduced in kernel v5.6.
+    #[doc(alias = "BPF_MAP_TYPE_STRUCT_OPS")]
+    StructOps = bpf_map_type::BPF_MAP_TYPE_STRUCT_OPS as isize,
+    /// A Ring Buffer map type. See [`RingBuf`](super::ring_buf::RingBuf) for the map
+    /// implementation.
+    ///
+    /// Introduced in kernel v5.8.
+    #[doc(alias = "BPF_MAP_TYPE_RINGBUF")]
+    RingBuf = bpf_map_type::BPF_MAP_TYPE_RINGBUF as isize,
+    /// An Inode Storage map type.
+    ///
+    /// Introduced in kernel v5.10.
+    #[doc(alias = "BPF_MAP_TYPE_INODE_STORAGE")]
+    InodeStorage = bpf_map_type::BPF_MAP_TYPE_INODE_STORAGE as isize,
+    /// A Task Storage map type.
+    ///
+    /// Introduced in kernel v5.11.
+    #[doc(alias = "BPF_MAP_TYPE_TASK_STORAGE")]
+    TaskStorage = bpf_map_type::BPF_MAP_TYPE_TASK_STORAGE as isize,
+    /// A Bloom Filter map type. See [`BloomFilter`](super::bloom_filter::BloomFilter) for the map
+    /// implementation.
+    ///
+    /// Introduced in kernel v5.16.
+    #[doc(alias = "BPF_MAP_TYPE_BLOOM_FILTER")]
+    BloomFilter = bpf_map_type::BPF_MAP_TYPE_BLOOM_FILTER as isize,
+    /// A User Ring Buffer map type.
+    ///
+    /// Introduced in kernel v6.1.
+    #[doc(alias = "BPF_MAP_TYPE_USER_RINGBUF")]
+    UserRingBuf = bpf_map_type::BPF_MAP_TYPE_USER_RINGBUF as isize,
+    /// A cGroup Storage map type.
+    ///
+    /// Introduced in kernel v6.2.
+    #[doc(alias = "BPF_MAP_TYPE_CGRP_STORAGE")]
+    CgrpStorage = bpf_map_type::BPF_MAP_TYPE_CGRP_STORAGE as isize,
+    /// An Arena map type.
+    ///
+    /// Introduced in kernel v6.9.
+    #[doc(alias = "BPF_MAP_TYPE_ARENA")]
+    Arena = bpf_map_type::BPF_MAP_TYPE_ARENA as isize,
+    /// An invalid map type.
+    #[doc(hidden)]
+    Invalid = bpf_map_type::__MAX_BPF_MAP_TYPE as isize,
```

Comment:
If we do TryFrom then I don't think we need this?

Reply:
🫡

Reply:
I think you forgot to remove this?

Reply:
Yep, forgot to 🪓 that. 😓